policies/coding-style.md

```diff
@@ -234,27 +234,51 @@ changing their context lines.
 
 ## Chapter 4: Naming
 
-C is a Spartan language, and so should your naming be. Do not use long
-names like _ThisVariableIsATemporaryCounter_. Use a name like _tmp_, which
-is much easier to write, and not more difficult to understand.
+C is a Spartan language, and so should your naming be.
 
-Except when otherwise required, avoid mixed-case names.
+Local variable names should be short, and to the point. If you have
+some random integer loop counter, it should probably be called `i` or `j`.
 
-Do not encode the type into a name (so-called Hungarian notation).
+Avoid single-letter names when they can be visually confusing,
+such as `I` and `O`.
+Avoid other single-letter names unless they are telling in the given context.
+For instance, `m` for modulus and `s` for SSL pointers are fine.
+
+Calling a variable `loop_counter` is non-productive,
+if there is no chance of it being mis-understood.
```

Comment:
This is probably redundant as you already say use i of j above

Reply:
I agree - so removed this sentence since it is in some sense also covered by the next one:

> Use simple variable names like `tmp` and `name` as long as they are non-ambiguous in the given context.